src/ArduinoGraphics.cpp

```diff
@@ -269,38 +271,57 @@ void ArduinoGraphics::textFont(const Font& which)
 
 int ArduinoGraphics::textFontWidth() const
 {
-  return (_font ? _font->width : 0);
+  return (_font ? _font->width * _textsize_x : 0);
 }
 
 int ArduinoGraphics::textFontHeight() const
 {
-  return (_font ? _font->height : 0);
+  return (_font ? _font->height* _textsize_y : 0);
 }
 
+void ArduinoGraphics::setTextSize(uint8_t sx, uint8_t sy)
+{
+  _textsize_x = (sx > 0)? sx : 1;
+  _textsize_y = (sy > 0)? sy : 1;
+}
+
+
 void ArduinoGraphics::bitmap(const uint8_t* data, int x, int y, int width, int height)
```

Comment:
```suggestion
void ArduinoGraphics::bitmap(const uint8_t* data, int x, int y, int w, int h, uint8_t scale_x, uint8_t scale_y)
```

Reply:
I believe I pushed up the changes that were requested.
First attempt forgot to update the example sketch.

Let me know if I missed anything.